src/ui/Tab/index.ts

```diff
@@ -0,0 +1,3 @@
+export { default as TabGroup } from './components/TabGroup';
```

Comment:
So what would be the alternative for exporting only names we desire from a module? In the case, we only want to export `Tab`, `TabGroup` and `useTabGroup()` from the `@ui/Tab` module, everything else should be used internally of this module only. Using `TabButton` from `@ui/Tab/components/TabButton` outside of a `TabGroup`, for example, will cause an error because it makes use of the `TabOptionsContext`.

Besides, I actaully don't notice any slowing down in Intellisense at all. I think after the first caching, TS should be able to provide Intellisense just fine. Don't think the slow Intellisense should be the reason to not use this perfect feature for code modulization.

Reply:
Then may as well refactor the other components and modules to index.ts (e.g. /modules/services, /mobules, etc.)

Reply:
Yea I think we should as well. But it's not a prioritized task since most of them do not have multiple files/components supposed to be used only internally.

(Should do this before documentation tho)